gatsby-config.js

```diff
@@ -67,6 +69,7 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-mdx',
       options: {
+        remarkPlugins: [indentedCodeBlock],
```

Comment:
why is this added here?

Reply:
Unfortunately [there is an issue](https://github.com/mdx-js/mdx/issues/1283) with MDX v2 that causes indented code blocks from parsing correctly. Whether or not this is intentional, it means that indented code blocks output to the screen with the backticks in the output.

Because its a _parsing_ issue, this means that a simple codemod won't work to solve this issue. Our `indentedCodeBlock` codemod also doubles as a remark plugin, so I've added this hear to patch the issue so that we can have indented code blocks rendering as we expect. Check out that GitHub issue if you want a deeper dive into the issue.

Reply:
oooh gotcha! thank you